src/run_program.rs

```diff
@@ -763,7 +743,7 @@ const TEST_CASES: &[RunProgramTest] = &[
         flags: 0,
         result: None,
         cost: 0,
-        err: "in ((X)...) syntax X must be lone atom",
+        err: "((X)...) syntax takes exactly 1 argument",
```

Comment:
This message isn't quite right. The idea it's trying to get across is that `X` must be an atom and not a cons. It can take multiple arguments (which are implicitly quoted).

Example:

`((+) 1 2 3) = 6`

We should make sure there is at least one test of this.

Reply:
The intended use for this is for a clvm implementation of a clvm emulator. It would descend trees and evaluated subtrees. It evaluated arguments first, and when it sees an operator it wants to pass through, it would package it up into `(op)` then cons it to the operator list yielding something like `((+) 1 2 3)`. Then you "apply" with `a` and you get `6`.

Reply:
Without this trick, you'd have to iteratively quote the arguments to build up `(+ (q . 1) (q . 2) (q . 3))` which scales per operators, and would slow things down quite a lot.